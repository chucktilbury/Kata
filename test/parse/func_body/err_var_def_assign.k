;-------------------------------------------------------------------------------
; A var def assignment requires an expression or an assignemnt item
;-------------------------------------------------------------------------------

private variable compound.name varname = dict
