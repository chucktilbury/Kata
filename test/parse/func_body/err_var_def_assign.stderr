Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_var_def_assign.k: 6: 1: expected an assignment item but got dict
